### PR TITLE
Feat : Alert and button

### PR DESCRIPTION
**Pull request by anuj-kumary**
Alert component added

https://user-images.githubusercontent.com/65344146/152336050-5248ca15-b8ed-4988-8359-6618d27cf170.mp4

### CODE REVIEW

**Review by thegeorgenikhil — APPROVED** [review]
Really good naming👍
Looks good to merge!

**Review by romabulani — APPROVED** [review]
Good to merge

**Review by ravikumar1002 — APPROVED** [review]

File: components/Button/button.css [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+    background-color:var(--off-white);
+  }
+  .outlined {
+    background-color: var(--white);
+    border: 1px solid var(--primary-color);
+    color: var(--primary-color);
+  }
+  .outlined__secondary {
+    background-color: var(--white);
+    border: 1px solid var(--black);
```
Comment (Hnh013) [inline components/Button/button.css]: use utility class for border width and style or add in base class

Comment (anuj-kumary) [reply components/Button/button.css]: made changes add the border in the base class

**Review by hrishi82 — APPROVED** [review]
Approved

File: components/Alert/alert.css [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    align-items: center;
+  }
+  .alert__success,
+  .alert__info,
+  .alert__error,
+  .alert__warning {
+    color: var(--success-color);
+    margin: 1rem;
+    padding: 1rem;
+    background-color: #edf7ec;
```
Comment (Parul1999) [inline components/Alert/alert.css]: CSS variable

File: components/Alert/alert.html [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Eagle | Alert</title>
+    <link rel="stylesheet" href="/css/component.css">
+</head>
+<body>
+    <div class="alert">
+        <div class="alert__success">
+          <span><i class="far fa-check-circle"></i></span>
+          <strong>Success!! </strong>A simple success alert—check it out!
```
Comment (Parul1999) [inline components/Alert/alert.html]: Can enclose text outside strong in a tag

File: components/Badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+  }
+  .badge__avatar-lg {
+    height: 8rem;
+    width: 8rem;
+    border-radius: 50%;
+  }
+  .status-online {
+    background-color: #1b7052;
+    position: absolute;
+    bottom: 4%;
```
Comment (Parul1999) [inline components/Badge/badge.css:26]: Maintain consistency in units . As bottom is in % , left is in rem

File: components/Button/button.css [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+    --white:#fff;
+    --black:#111;
+    --blue: #0000ff;
+    --off-white: #f8fafc;
+    --border: 1px solid;
+}
+.btn {
+    padding: 0.5rem 1rem;
+    font-weight: 600;
+    min-width: 6.25;
```
Comment (Parul1999) [inline components/Button/button.css]: put units

File: components/Alert/alert.css (hunk 2 above)
Comment (anuj-kumary) [reply components/Alert/alert.css]: Done

**Review by Rohanmond — APPROVED** [review]
looks good

**Review by Parul1999 — APPROVED** [review]

**Review by Hnh013 — COMMENTED** [review]
looks good with changes !

**Pull request merged**